### ISSUE TITLE
[Accessibility] Localize the "This is the single point tooltip template" string in the activity-timeline.component.html file to English

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4989]
# Summary
## Describe the bug
The string **`"This is the single point tooltip template"`** on line 23 in the activity-timeline.component.html file needs to be localized to support the English language.

## To Reproduce
Navigate to the `apps/web-mzima-client/src/app/activity/activity-timeline/activity-timeline.component.html` file.
Locate the following code block on line 23:

```html
<ng-template #tooltipTemplate let-model="model">
  This is the single point tooltip template
  <pre>{{ model | json }}</pre>
</ng-template>
```
## Expected behavior
The "This is the single point tooltip template" string should be localized and retrieved from a translation file or service, so that it can be displayed in English.

## How to solve
- In the `activity-timeline.component.html` file, replace the hardcoded "This is the single point tooltip template" string with a translation key (something like., {{ 'activity.tooltip_template' | translate }}).
- Add the translation key-value pair in the en.json file in the` apps/web-mzima-client/src/assets/locales/ `directory:

```
{
  "activity": {
    "tooltip_template": "This is the single point tooltip template"
  }
}
```
Screenshots
<img width="1200" alt="Screenshot 2024-07-26 at 14 57 11" src="https://github.com/user-attachments/assets/07d84158-f474-4c60-9bef-6ea401f51b33">
